styles.css

```diff
@@ -34,34 +34,60 @@ main {
 }
 
 section {
-  border: solid 2px darkred;
+  border: solid 4px darkred;
   display: flex;
   flex-direction: column;
   width: 90%;
   margin: auto;
   padding: 20px 20px;
-  align-items: center;
   list-style: none;
 }
 
 ul {
   display: flex;
   flex-direction: row;
```

Comment:
`flex-direction: row` is the default behavior of flexbox and therefore you do not need to declare it.